Traceback (most recent call last):
  File "/scr1/sumo/tests/../tools/generateParkingAreaRerouters.py", line 455, in <module>
    main(sys.argv[1:])
  File "/scr1/sumo/tests/../tools/generateParkingAreaRerouters.py", line 450, in main
    ReroutersGeneration(args)
  File "/scr1/sumo/tests/../tools/generateParkingAreaRerouters.py", line 152, in __init__
    self._load_parking_areas_from_file(pafile)
  File "/scr1/sumo/tests/../tools/generateParkingAreaRerouters.py", line 171, in _load_parking_areas_from_file
    laneID = child.attrib['lane']
KeyError: 'lane'
